tools/d2g/d2g.go

```diff
@@ -30,8 +30,9 @@ type (
 	DockerImageArtifact dockerworker.DockerImageArtifact
 	Image               interface {
 		FileMounts() ([]genericworker.FileMount, error)
-		String() (string, error)
+		String() string
 		LoadCommands() []string
+		ChainOfTrustCommands() []string
```

Comment:
Could make two helpers for forming the two strings returned for this method to prevent the duplicated code, potentially helping us from future issues.